app/client/ui/errorPages.ts

```diff
@@ -105,7 +105,7 @@ export function createNotFoundPage(appModel: AppModel, message?: string) {
     })),
     cssButtonWrap(bigPrimaryButtonLink(t("Go to main page"), testId('error-primary-btn'),
       urlState().setLinkUrl({}))),
-    cssButtonWrap(bigBasicButtonLink(t("Contact support"), {href: getGristConfig().contactSupportUrl})),
+    cssButtonWrap(bigBasicButtonLink(t("Contact support"), {href: getContactSupportUrl()})),
```

Comment:
```suggestion
    cssButtonWrap(bigBasicButtonLink(t("Contact support"), {href: commonUrls.contact})),
```
You can do the same here and just update the value of that key to be `getContactSupportUrl()`.